pkg/kubernetes/templates/v1beta2/kubeadm_config.go

```diff
@@ -311,10 +311,12 @@ func GetKubeletCgroupDriver(runtime connector.Runtime, kubeConf *common.KubeConf
 	if err != nil {
 		return "", errors.Wrap(errors.WithStack(err), "Failed to get container runtime cgroup driver.")
 	}
-	if strings.Contains(checkResult, "systemd") || !strings.Contains(checkResult, "false") {
+	if strings.Contains(checkResult, "systemd") || strings.Contains(checkResult, "SystemdCgroup = true") {
```

Comment:
The result of this command is:
```
$ containerd config dump | grep SystemdCgroup | awk -F'= ' '{ print $2; }'
true

$ containerd config dump | grep SystemdCgroup | awk -F'= ' '{ print $2; }'
false
```
So this case `strings.Contains(checkResult, "SystemdCgroup = true")` will always be `false`.

Reply:
got it let me fix